### ISSUE TITLE
Cannot install any games anymore

### ISSUE THREAD

**Issue by t0msk** [lutris/lutris#4255]
### Bug description

I tried to install Elite Dangerous using Proton version and it hangs.

### How to Reproduce

Install Elite Dangerous using Proton version.

### Expected behavior

Games should be installed

### Log output

```shell
[tomsk@tomsk-PC ~]$ lutris -d
INFO     2022-05-02 16:20:53,032 [startup.init_lutris:173]:Starting Lutris 0.5.10.1
WARNING  2022-05-02 16:20:53,065 [libretro.get_libretro_cores:24]:No folder at /home/tomsk/.local/share/lutris/runners/retroarch/
INFO     2022-05-02 16:20:53,099 [startup.check_driver:57]:Using NVIDIA drivers 510.60.02 for x86_64
INFO     2022-05-02 16:20:53,099 [startup.check_driver:61]:GPU: NVIDIA GeForce GTX 1060 6GB
INFO     2022-05-02 16:20:53,099 [startup.check_driver:77]:GPU: 10DE:1C03 1458:3716 (nvidia drivers)
INFO     2022-05-02 16:20:53,100 [startup.check_driver:77]:GPU: 8086:0412 1043:8534 (i915 drivers)
MESA-INTEL: warning: Haswell Vulkan support is incomplete
INFO     2022-05-02 16:20:53,155 [startup.update_runtime:213]:Startup complete
DEBUG    2022-05-02 16:20:53,255 [lutriswindow.update_store:437]:Showing 50 games
DEBUG    2022-05-02 16:20:54,787 [application.show_window:332]:Showing window AddGamesWindow{}
DEBUG    2022-05-02 16:21:00,888 [http.get:83]:GET https://lutris.net/api/games?search=elite+dangerous&with-installers=True
DEBUG    2022-05-02 16:21:02,100 [api.get_game_installers:160]:Fetching installer https://lutris.net/api/installers/elite-dangerous
DEBUG    2022-05-02 16:21:02,100 [http.get:83]:GET https://lutris.net/api/installers/elite-dangerous
DEBUG    2022-05-02 16:21:02,380 [application.show_window:332]:Showing window InstallerWindowelite-dangerous
DEBUG    2022-05-02 16:21:02,381 [application.on_app_window_destroyed:350]:Removed window AddGamesWindow{}
DEBUG    2022-05-02 16:21:04,840 [displayconfig.get_current_resolution:624]:Retrieving current resolution
ERROR    2022-05-02 16:21:04,840 [displayconfig.get_current_resolution:627]:Could not retrieve the current display mode
DEBUG    2022-05-02 16:21:04,843 [interpreter.get_extras:159]:Checking if service provide extra files
INFO     2022-05-02 16:21:04,843 [installer.prepare_game_files:143]:No files to prepare
DEBUG    2022-05-02 16:21:04,843 [installerwindow.on_runners_ready:312]:Installer doesn't require files
WARNING  2022-05-02 16:21:04,843 [interpreter.launch_installer_commands:282]:Installer with only extras and no game files
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/lutris/gui/installerwindow.py", line 313, in on_runners_ready
    self.interpreter.launch_installer_commands()
  File "/usr/lib/python3.10/site-packages/lutris/installer/interpreter.py", line 289, in launch_installer_commands
    self._iter_commands()
  File "/usr/lib/python3.10/site-packages/lutris/installer/interpreter.py", line 320, in _iter_commands
    self._finish_install()
  File "/usr/lib/python3.10/site-packages/lutris/installer/interpreter.py", line 344, in _finish_install
    game_id = self.installer.save()
  File "/usr/lib/python3.10/site-packages/lutris/installer/installer.py", line 253, in save
    configpath = write_game_config(self.slug, self.get_game_config())
  File "/usr/lib/python3.10/site-packages/lutris/installer/installer.py", line 229, in get_game_config
    config["game"] = self._substitute_config(config["game"])
  File "/usr/lib/python3.10/site-packages/lutris/installer/installer.py", line 196, in _substitute_config
    config[key] = self.interpreter._substitute(value)
  File "/usr/lib/python3.10/site-packages/lutris/installer/interpreter.py", line 420, in _substitute
    return system.substitute(template_string, self._get_string_replacements())
  File "/usr/lib/python3.10/site-packages/lutris/installer/interpreter.py", line 400, in _get_string_replacements
    "RESOLUTION_WIDTH_HEX": hex(int(self.current_resolution[0])),
ValueError: invalid literal for int() with base 10: ''
```

### System Information

```shell
[System]
OS:              Arch Linux rolling n/a
Arch:            x86_64
Kernel:          5.17.5-arch1-1
Desktop:         GNOME
Display Server:  x11

[CPU]
Vendor:          GenuineIntel
Model:           Intel(R) Core(TM) i5-4670 CPU @ 3.40GHz
Physical cores:  4
Logical cores:   4

[Memory]
RAM:             15.3 GB
Swap:            7.6 GB

[Graphics]
Vendor:          Intel
OpenGL Renderer: Mesa Intel(R) HD Graphics 4600 (HSW GT2)
OpenGL Version:  3.1 Mesa 22.0.2
OpenGL Core:     4.6 (Core Profile) Mesa 22.0.2
OpenGL ES:       OpenGL ES 3.2 Mesa 22.0.2
Vulkan:          Supported
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by danieljohnson2** [comment]
This looks like the storied issue #4218.

If so, there's an outstanding PR #4228, which is from [this branch](https://github.com/danieljohnson2/lutris/tree/dj/monitoring_the_monitors).

Give that a try and see if that doesn't help. I'm hoping that'll wind up in a point release before too long, but I expect that won't happen without a resolution to #4235.

**Comment by strycore** [comment]
This has been patched so marking as fixed in 0.5.11
